js/actions/appActions.js

```diff
@@ -1837,9 +1837,10 @@ const appActions = {
     })
   },
 
-  onPromotionResponse: function () {
+  onPromotionResponse: function (error) {
     dispatch({
-      actionType: appConstants.APP_ON_PROMOTION_RESPONSE
+      actionType: appConstants.APP_ON_PROMOTION_RESPONSE,
+      error
```

Comment:
Are we wanting to call this error? It seems like this also contains the success response?

Reply:
renamed